.golangci.yml

```diff
@@ -19,7 +19,7 @@ linters:
     - gofumpt # "gofmt" is OK for us, no need for stricter rules (https://github.com/mvdan/gofumpt)
     - gci # No need to explicitly format imports using GCI (https://github.com/daixiang0/gci)
     - depguard # No need to restrict imported packages (https://github.com/OpenPeeDeeP/depguard)
-    - goerr113 # Using static errors is not necessary (https://github.com/Djarvur/go-err113)
+    - err113 # Using static errors is not necessary (https://github.com/Djarvur/go-err113)
```

Comment:
Note: upgrade of linter required changes in this config file and added new warnings. I fixed them all.